client/src/areas/party/UserManagerRenderer.tsx

```diff
@@ -0,0 +1,216 @@
+import * as React from "react";
+import Table, { TableProps } from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper, { PaperProps } from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography/Typography";
+import styled from "styled-components";
+import { UserManagementButton } from "./components/UserManagementButton";
+import { PartyManager } from "./PartyManager";
+import { IPartyUserData } from "../../common/definitions/union/IPartyUserData";
+
+export interface IUserManagerRendererProps {
+  users: IPartyUserData;
+}
+
+interface IUserManagerState {
+  users: IPartyUserData;
+  isLoading: boolean;
+}
+
+export class UserManagerRenderer extends React.Component<
+  IUserManagerRendererProps,
+  IUserManagerState
+> {
+  public constructor(props: IUserManagerRendererProps) {
+    super(props);
+    this.state = {
+      users: props.users,
+      isLoading: false
+    };
+  }
+
+  public render() {
```

Comment:
When you refresh the page on the manage page, and in the mean time somebody new requested to join, you have to reload again (so 2 x  times in total) to see the request

Reply:
#32